### PR TITLE
added output

### PR DESCRIPTION
**Pull request by andrewprakash**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
-      shell: bash
-    - run: ls -al
-      shell: bash
-    - run: docker build -t sencha-app --build-arg senchaVersion=${{inputs.sencha-version}} --output ./${{inputs.build-folder}} .
-      shell: bash
-    - run: tar -zcvf ${{inputs.build-folder}}.tar.gz ${{inputs.build-folder}}
-      shell: bash
-    - run: aws s3 cp ${{inputs.build-folder}}.tar.gz s3://${{inputs.bucket-with-prefixes}}/2.3.${{ github.run_number }}
-      shell: bash
+  using: 'node12'
```
Comment (tfentonz) [inline action.yml]: Can we use Node.js 16 Active Long Term Support (LTS)?

**Review by tfentonz — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (andrewprakash) [reply action.yml]: done.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": true,
+    "printWidth": 80,
+    "singleQuote": true
+}
```
Comment (tfentonz) [inline .prettierrc]: Add lint and lint:fix to scripts in `packages.json`

```js
    "lint": "eslint \"**/*.ts\" --ignore-pattern \"**/*.d.ts\" --quiet",
    "lint:fix": "eslint \"**/*.ts\" --ignore-pattern \"**/*.d.ts\" --fix --quiet"
```

Add `.eslintrc` file. See cdk-template [.eslintrc](https://github.com/magiqsoftware/cdk-template/blob/main/.eslintrc.js).

Can then run `npm run lint`.

**Review by tfentonz — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 2 above)
Comment (andrewprakash) [reply .prettierrc]: done. https://github.com/magiqsoftware/enterprise-frontend-action/pull/1/commits/17a4c989dd08e935a634da2c4ac93e554c846e47

**Review by tfentonz — APPROVED** [review]

**Pull request merged**